src/content/reducers/url-state.js

```diff
@@ -147,3 +148,8 @@ export const getURLPredictor = createSelector(
     return urlFromState(newURLState);
   }
 );
+
+export const getRangeFilterLabels = createSelector(
```

Comment:
It would be nice if we could have this memoization inside the connected component without having to add another selector, and with the same perf advantages. Not sure how to easily do that, though.